src/iris/bin/sender.py

```diff
@@ -1084,7 +1086,22 @@ def mock_gwatch_renewer():
         sleep(60)
 
 
+def sender_shutdown():
+    logger.info('Shutting server..')
+
+    # Immediately release all locks and give up any master status and slave presence
+    if coordinator:
+        coordinator.leave_cluster()
+
+    # Force quit. Avoid sender process existing longer than it needs to
+    os._exit(0)
```

Comment:
Currently, this will not save any existing OOB messages in the queue. It shouldn't affect messages coming from incidents though as another sender will take on continuing incident steps.